device/esp_tinyusb/tinyusb.c

```diff
@@ -69,5 +69,6 @@ esp_err_t tinyusb_driver_install(const tinyusb_config_t *config)
 esp_err_t tinyusb_driver_uninstall()
 {
     tinyusb_free_descriptors();
+    tusb_teardown();
```

Comment:
@tore-espressif ,

*when you will have time*

Is there any good way how I can make this change backward compatible with previous `espressif/tinyusb` components (previous versions, which doesn't have this implementation)? 
Will appreciate any suggestions.

Reply:
Here are some ideas, that came to me: 
1. Tie it to the configuration and use idf version to determine the necessity and implementation of tusb_teardown()
2. Define it somewhere in esp_tinyusb with WEAK parameter. Which let us redefine in with new version and don't panic with old versions, when there is no implementation of it
3. Enable it with menuconfig (which seems really strange for me, but anyway maybe it is possible).

I like option No.2, but maybe there are other simple and better solution.